lib/commands/general.js

```diff
@@ -13,24 +13,26 @@ commands.getWindowSize = async function getWindowSize () {
   }
   // workaround for https://github.com/microsoft/WinAppDriver/issues/1104
   log.info('Cannot retrieve window size from WinAppDriver. ' +
-    'Falling back to WMIC usage');
-  const {stdout} = await exec('wmic.exe', [
-    'path', 'Win32_VideoController',
-    'get', 'VideoModeDescription,CurrentVerticalResolution,CurrentHorizontalResolution',
-    '/format:value',
-  ]);
-  log.debug(stdout);
-  const widthMatch = /CurrentHorizontalResolution=(\d+)/.exec(stdout);
-  const heightMatch = /CurrentVerticalResolution=(\d+)/.exec(stdout);
+      'Falling back to Windows Forms to calculate dimensions');
+  const widthMatch = await getDimension('Width');
+  const heightMatch = await getDimension('Height');
   if (!widthMatch || !heightMatch) {
     throw new Error('Cannot retrieve the screen size. Check the server log for more details');
   }
   return {
-    width: parseInt(widthMatch[1], 10),
-    height: parseInt(heightMatch[1], 10),
+    width: parseInt(widthMatch, 10),
+    height: parseInt(heightMatch, 10),
   };
 };
 
+async function getDimension (dimensionType) {
+  const {stdout} = await exec('powershell', [
+    '-command', 'Add-Type -AssemblyName System.Windows.Forms;[System.Windows.Forms.Screen]::PrimaryScreen.Bounds.Size.' + dimensionType,
```

Comment:
Please use template strings instead of catenation: https://developer.mozilla.org/en-US/docs/Web/JavaScript/Reference/Template_literals